portafly/src/pages/DeveloperAccounts.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useDocumentTitle, Loading, DeveloperAccountsTable } from 'components'
+import {
+  Alert,
+  PageSection,
+  PageSectionVariants,
+  TextContent,
+  Text
+} from '@patternfly/react-core'
+import { useGetDeveloperAccounts } from 'dal/accounts'
+import { useTranslation } from 'i18n/useTranslation'
+
+const DeveloperAccounts: React.FunctionComponent = () => {
+  const { t } = useTranslation('accounts')
+  useDocumentTitle(t('page_title'))
+
+  const { accounts, error, isPending } = useGetDeveloperAccounts()
+
+  return (
+    <>
+      <PageSection variant={PageSectionVariants.light}>
+        <TextContent>
+          <Text component="h1">{t('body_title')}</Text>
+        </TextContent>
+      </PageSection>
+
+      <PageSection>
+        {isPending && <Loading />}
+        {error && <Alert variant="danger" title={error.message} />}
+        {accounts && <DeveloperAccountsTable accounts={accounts} />}
+      </PageSection>
+    </>
+  )
+}
+
+// Default export needed for React.lazy
```

Comment:
If we are disabling this rule so often, maybe we could just remove it. I know it's out of scope of this PR, just keep it in mind.

Reply:
The rest of the project should follow the rule so I would not disable it, it's only skipped in `pages/`. We could however add an `eslint` file inside then folder that overrides the rule, to avoid verbosity.